lua/plenary/strings.lua

```diff
@@ -0,0 +1,165 @@
+local path = require('plenary.path').path
+
+local M = {}
+
+M.strdisplaywidth = (function()
+  if jit and path.sep ~= [[\]] then
+    local ffi = require('ffi')
+    ffi.cdef[[
+      typedef unsigned char char_u;
+      int linetabsize_col(int startcol, char_u *s);
+    ]]
+
+    return function(str, col)
+      local startcol = col or 0
+      local s = ffi.new('char[?]', #str + 1)
+      ffi.copy(s, str)
+      return ffi.C.linetabsize_col(startcol, s) - startcol
+    end
+  else
+    return function(str, col)
```

Comment:
didn't we add a `tostring` to this one as well (in telescope for some reason?)

Reply:
Fixed on bbf658a referring to https://github.com/nvim-telescope/telescope.nvim/pull/722#discussion_r609602563